shared/settings/delete-confirm/container.js

```diff
@@ -2,14 +2,30 @@
 import React, {Component} from 'react'
 import {TypedConnector} from '../../util/typed-connect'
 import {navigateUp} from '../../actions/router'
+import {HOCTimers} from '../../common-adapters'
 import DeleteConfirm from './index'
-import {deleteAccountForever} from '../../actions/settings'
+import {setAllowDeleteAccount, deleteAccountForever} from '../../actions/settings'
 
 import type {TypedDispatch} from '../../constants/types/flux'
 import type {TypedState} from '../../constants/reducer'
+import type {TimerProps} from '../../common-adapters/hoc-timers'
 import type {Props} from './index'
 
-class DeleteConfirmContainer extends Component<void, Props, void> {
+class DeleteConfirmContainer extends Component<void, Props & TimerProps, void> {
+  componentWillMount () {
```

Comment:
ah. my thinking was this would be set by the previous screen before it navigated to this one. but this could also work. my concern was you didn't see the last screen. 👍  either way